GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Atlas.cfg

```diff
@@ -403,22 +422,11 @@
     MODULE
     {
         name = ModuleFuelTanks
-        type = ServiceModule
+        type = BalloonCryo
```

Comment:
Ahh, right, I remember reading those docs long ago. Yeah, BalloonCryo should be good then. Possibly a future review to compare the on orbit staying time of our configs to the real ones will be in order to confirm or lead us to fine tune such things.